whois-api/src/main/java/net/ripe/db/whois/api/rest/RestServiceHelper.java

```diff
@@ -65,14 +67,18 @@ private static String filter(final String queryString) {
         final StringBuilder builder = new StringBuilder();
         char separator = '?';
 
-        for (String next : AMPERSAND_SPLITTER.split(queryString)) {
-            final Iterator<String> iterator = EQUALS_SPLITTER.split(next).iterator();
-            if (iterator.hasNext() && iterator.next().equalsIgnoreCase("password")) {
-                continue;
-            }
+        if (queryString.contains(OVERRIDE_STRING)) {
```

Comment:
Thought this morning: consider moving  this logic into `PasswordFilter.filterPasswordsInUrl`. Before merging see if that won't be a better way to have it

Reply:
👍

Reply:
This will involve passing in a `flag` parameter to `PasswordFilter.filterPasswordsInUrl` method, to tell it whether the url should be filtered for use in response body (then password query and value is removed) or whether the query should be filtered for audit logs (then value of password is replaced with `FILTERED`).    

This is because there is nothing in the log for the method to decide this.

Hence I will leave as is, as I agree with Fowler, [Flag arguments should generally be avoided](https://martinfowler.com/bliki/FlagArgument.html)